### PR TITLE
Completed Page LC and Http Handler Migrations

### PR DESCRIPTION
**Pull request by comsemer**
## Description
- All page lifecycle events being migrated
- Http Handlers being migrated
- Added ComponentSyntaxHelper to build component lc methods
- Also slightly modified LifecycleManagerService

---
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

**Review by jonlouie — APPROVED** [review]
LGTM

File: src/CTA.WebForms2Blazor/Extensions/SemanticModelExtensions.cs [hunk 1]
```diff
@@ -53,9 +53,10 @@ public static IEnumerable<INamespaceSymbol> GetNamespacesReferencedByType(this S
             // to as references within the same namespace are already accessible
             namespaceSymbols.Remove(classTypeSymbol.ContainingNamespace);
 
-            // TODO: Find out why null namespaces occur and maybe
-            // replace with Codelyzer usage
-            return namespaceSymbols.Where(namespaceSymbol => namespaceSymbol != null);
+            // TODO: Find out why null namespaces occur and maybe replace with Codelyzer usage
+            // We need to remove occurrences of the global namespace, it seomtimes gets added
```
Comment (Eruanion) [inline src/CTA.WebForms2Blazor/Extensions/SemanticModelExtensions.cs:57]: Tiny typo on 'sometimes'

File: src/CTA.WebForms2Blazor/Helpers/ComponentSyntaxHelper.cs [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+﻿using System.Collections.Generic;
 using System.Linq;
-using Microsoft.CodeAnalysis;
+using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 
 namespace CTA.WebForms2Blazor.Helpers
 {
     public static class ComponentSyntaxHelper
```
Comment (Eruanion) [inline src/CTA.WebForms2Blazor/Helpers/ComponentSyntaxHelper.cs:9]: Just a nitpick a lot of the methods use the word Build in front of them which carries a meaning that somehow the method will actually 'build' a project somehow. I tihnk the better meaning here is probably construct and for BuildOn maybe AddTo.

**Review by Eruanion — COMMENTED** [review]
Just a couple of comments, good work :-)

**Review by Eruanion — APPROVED** [review]

**Review by KallanBao — APPROVED** [review]

**Pull request merged**